### DISCUSSION TITLE
Unable to execute binary files in Ubuntu 22.04 ARM64

### DISCUSSION THREAD

**Discussion by payalwad** [utmapp/UTM#6131] [Linux]
**Describe the issue**  
Hi, I am unable to run binary executables on Ubuntu 22.04 ARM64. I am getting this error
x86_64-binfmt-P: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory

Please help me with the same. Thanks in advance!

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: Ventura 13.3
* Mac Chip (Intel, M1, ...): M2

**Comment by gfrankliu** [comment]
It seems you are trying to run x86 binary. At minimum, you would need to install the x86 libc:

```
sudo dpkg --add-architecture amd64 
sudo apt update
sudo apt install libc6:amd64
```

This assumes you already have rosetta setup. Otherwise, you need to follow https://docs.getutm.app/advanced/rosetta/
